### PR TITLE
Pioneer 17 surface moorings

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP03ISSM/CP03ISSM_D00014_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00014/cg_data/dcl11/mopak/*.mopak.log,CP03ISSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will be available upon recovery
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00014/cg_data/dcl11/hyd1/*.hyd1.log,CP03ISSM-SBD11-02-HYDGN0000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00014/cg_data/dcl11/metbk/*.metbk.log,CP03ISSM-SBD11-06-METBKA000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00014/cg_data/dcl12/syslog/*.syslog.log,CP03ISSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00014/cg_data/dcl12/hyd2/*.hyd2.log,CP03ISSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/D00014/cg_data/dcl12/pco2a/*.pco2a.log,CP03ISSM-SBD12-04-PCO2AA000,telemetered,Available,
```
Comment (asmith-whoi) [inline CP03ISSM/CP03ISSM_D00014_ingest.csv:32]: OK

File: CP04OSSM/CP04OSSM_D00014_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl11/mopak/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will be available upon recovery
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl11/hyd1/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl11/metbk/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl12/hyd2/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl12/pco2a/*.pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,telemetered,Available,
```
Comment (asmith-whoi) [inline CP04OSSM/CP04OSSM_D00014_ingest.csv:32]: OK

File: CP01CNSM/CP01CNSM_D00015_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl12/syslog/*.syslog.log,CP01CNSM-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl12/hyd2/*.hyd2.log,CP01CNSM-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl12/pco2a/*.pco2a.log,CP01CNSM-SBD12-04-PCO2AA000,telemetered,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl12/wavss/*.wavss.log,CP01CNSM-SBD12-05-WAVSSA000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl12/metbk2/*.metbk2.log,CP01CNSM-SBD12-06-METBKA000,telemetered,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl12/fdchp/*.fdchp.log,CP01CNSM-SBD12-08-FDCHPA000,telemetered,Available,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply CP01CNSM/CP01CNSM_D00015_ingest.csv:35]: Oh good catch, Allen!  Yup - I concur!

File: CP01CNSM/CP01CNSM_D00014_ingest.csv [hunk 4]
```diff
@@ -24,7 +24,7 @@ mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/who
 mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00014/cg_data/dcl27/dosta1/*dosta1.log,CP01CNSM-RID27-04-DOSTAD000,telemetered,Available,
 mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00014/irid2shore/*cpm_status*.txt,CP01CNSM-SBC11-00-CPMENG000,telemetered,Available,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00014/cg_data/dcl11/syslog/*syslog.log,CP01CNSM-SBD11-00-DCLENG000,telemetered,Available,
-#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00014/cg_data/dcl11/mopak/*mopak.log,CP01CNSM-SBD11-01-MOPAK0000,Not Expected,Data will be available upon recovery,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00014/cg_data/dcl11/mopak/*mopak.log,CP01CNSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will be available upon recovery
```
Comment (rtravis-whoi) [inline CP01CNSM/CP01CNSM_D00014_ingest.csv:27]: looks good

File: CP01CNSM/CP01CNSM_D00015_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl27/flort/*.flort.log,CP01CNSM-RID27-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl27/ctdbp1/*.ctdbp1.log,CP01CNSM-RID27-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl27/dosta1/*.dosta1.log,CP01CNSM-RID27-04-DOSTAD000,telemetered,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/irid2shore/cpm_status*.txt,CP01CNSM-SBC11-00-CPMENG000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl11/syslog/*.syslog.log,CP01CNSM-SBD11-00-DCLENG000,telemetered,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/D00015/cg_data/dcl11/mopak/*.mopak.log,CP01CNSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Data will be available upon recovery
```
Comment (rtravis-whoi) [inline CP01CNSM/CP01CNSM_D00015_ingest.csv:27]: correction looks good

File: CP01CNSM/CP01CNSM_D00015_ingest.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply CP01CNSM/CP01CNSM_D00015_ingest.csv:35]: I don't believe there should be a comma after the notes column so this update looks good (referencing Allen's earlier review comment).

File: CP04OSSM/CP04OSSM_D00014_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/syslog/*.syslog.log,CP04OSSM-MFD37-00-DCLENG000,telemetered,Not Expected,DCL failed on deployment
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/optaa2_decimated/*.optaa2.log,CP04OSSM-MFD37-01-OPTAAD000,telemetered,Not Deployed,Instrument not deployed
+#mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/ctdbp2/*.ctdbp2.log,CP04OSSM-MFD37-03-CTDBPE000,telemetered,Not Expected,DCL failed on deployment
+#mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/dosta2/*.dosta2.log,CP04OSSM-MFD37-04-DOSTAD000,telemetered,Not Expected,DCL failed on deployment
+#mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/zplsc/*.zplsc.log,CP04OSSM-MFD37-07-ZPLSCC000,telemetered,Not Expected,DCL failed on deployment
```
Comment (rtravis-whoi) [inline CP04OSSM/CP04OSSM_D00014_ingest.csv:13]: lines commented out for DCL37 failure look good; OPTAA removed from MFN & commented out - good.

**Review by rtravis-whoi — APPROVED** [review]

File: CP01CNSM/CP01CNSM_D00014_ingest.csv (hunk 4 above)
Comment (asmith-whoi) [inline CP01CNSM/CP01CNSM_D00014_ingest.csv:27]: OK

File: CP01CNSM/CP01CNSM_D00015_ingest.csv (hunk 5 above)
Comment (asmith-whoi) [inline CP01CNSM/CP01CNSM_D00015_ingest.csv:27]: OK

File: CP01CNSM/CP01CNSM_D00015_ingest.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (asmith-whoi) [reply CP01CNSM/CP01CNSM_D00015_ingest.csv:35]: Clarifying my comment above... there appeared to be an extra comma, not a missing one. The missing data_source was the underlying cause.

File: CP04OSSM/CP04OSSM_D00014_ingest.csv (hunk 6 above)
Comment (asmith-whoi) [inline CP04OSSM/CP04OSSM_D00014_ingest.csv:13]: OK

**Review by asmith-whoi — APPROVED** [review]
Approving following changes.

**Pull request merged**